packages/console/src/pages/Connectors/components/ConnectorRow/index.tsx

```diff
@@ -7,31 +7,53 @@ import Status from '@/components/Status';
 import { connectorTitlePlaceHolder } from '@/consts/connectors';
 
 import ConnectorName from '../ConnectorName';
+import * as styles from './index.module.scss';
 
 type Props = {
   type: ConnectorType;
   connector?: ConnectorDTO;
+  connectors?: ConnectorDTO[];
   onClickSetup?: () => void;
 };
 
-const ConnectorRow = ({ type, connector, onClickSetup }: Props) => {
+const ConnectorRow = ({ type, connector, connectors, onClickSetup }: Props) => {
   const { t } = useTranslation(undefined);
 
   return (
     <tr>
       <td>
-        <ConnectorName type={type} connector={connector} />
+        <ConnectorName
+          type={type}
+          connector={connector ?? connectors?.[0]}
+          isShowId={Boolean(connector)}
+        />
       </td>
       <td>{t(connectorTitlePlaceHolder[type])}</td>
       <td>
-        {type === ConnectorType.Social && (
-          <Status status={connector?.enabled ? 'enabled' : 'disabled'}>
-            {t(
-              connector?.enabled
-                ? 'admin_console.connectors.connector_status_enabled'
-                : 'admin_console.connectors.connector_status_disabled'
-            )}
-          </Status>
+        {type === ConnectorType.Social && connectors && (
+          <div className={styles.statusItems}>
+            {connectors.map(({ id, enabled, platform }) => (
+              <div key={id} className={styles.statusItem}>
+                <Status status={enabled ? 'enabled' : 'disabled'}>
+                  {t(
+                    enabled
+                      ? 'admin_console.connectors.connector_status_enabled'
+                      : 'admin_console.connectors.connector_status_disabled'
+                  )}
```

Comment:
This can be simplified. Tried the following and it worked

<img width="955" alt="image" src="https://user-images.githubusercontent.com/12833674/170908315-8e86e26c-b27f-4b0d-96c7-b76a3ffac5a0.png">

Reply:
updated